src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public  Random random = new Random();
+    public ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
+        String randomColor = colorSupplier.getRandomColor();  // отримуємо випадковий колір
+        int randomNumber = random.nextInt(101);  // випадкове число від 0 до 100 включно
```

Comment:
Magic number 101 is used here. It's better to define it as a constant to give it a meaningful name and to avoid magic numbers scattered throughout the code. For example, private static final int MAX_NUMBER = 101; Then you can use 'random.nextInt(MAX_NUMBER);'